packages/xo-web/src/xo-app/new/sr/index.js

```diff
@@ -56,7 +56,7 @@ class SelectScsiId extends Component {
     () => this.props.options,
     options =>
       map(options, ({ vendor, path, size, scsiId }) => ({
-        label: `${vendor} - ${path} (${formatSize(size)})`,
+        label: `${vendor} - ${path} (${formatSize(parseInt(size))})`,
```

Comment:
Please, use `+` prefix operator instead of `parseInt` which has some weird behaviors:

```js
parseInt('1a') // 1
+'1a' // NaN
```

Reply:
I've updated my pull request